pkg/cmd/sync/sync.go

```diff
@@ -100,13 +100,17 @@ func NewCmdSync(commonOpts *opts.CommonOptions) *cobra.Command {
 func (o *SyncOptions) Run() error {
 
 	// ksync is installed to the jx/bin dir, so we can add it for the user
-	os.Setenv("PATH", util.PathWithBinary())
+	err := os.Setenv("PATH", util.PathWithBinary())
+	if err != nil {
```

Comment:
+1, good catch

Reply:
I have an issue abt better error handling, and enabling errcheck as part of the lint step when we open a PR against jx master: https://github.com/jenkins-x/jx/issues/6851. I have a branch, where I am adding more error checks, it's a bigger work in progress atm.

Reply:
that would be nice